doc/releases/release-notes-3.5.rst

```diff
@@ -233,6 +233,23 @@ USB
 Devicetree
 **********
 
+Bindings
+========
+
+Modified bindings:
+
+  - Texas Instruments (ti):
+
+    - :dtcompatible:`ti,ina230` (on i2c bus):
+
+        - new property: ``rshunt-micro-ohms``
+        - removed property: ``rshunt-milliohms``
+
+    - :dtcompatible:`ti,ina237` (on i2c bus):
+
+        - new property: ``rshunt-micro-ohms``
+        - removed property: ``rshunt-milliohms``
```

Comment:
this is autogenerated before the release, please drop

Reply:
Removed.